src/crimson/os/seastore/backref/btree_backref_manager.h

```diff
@@ -34,17 +34,6 @@ class BtreeBackrefMapping : public BtreeNodeMapping<paddr_t, laddr_t> {
   extent_types_t get_type() const final {
```

Comment:
Looks `BtreeNodeMapping` has an implementation but with `ceph_abort()`.

Who uses `BtreeBackrefMapping::get_type()` ?

Reply:
It's used in `SegmentCleaner::do_reclaim_space()`. The cleaner use it to get extents.
https://github.com/ceph/ceph/blob/093e0de0b870f3cc10663e24dc60c991ac615612/src/crimson/os/seastore/async_cleaner.cc#L1140-L1146

Reply:
Yeah, will do.